sjb/config/common/test_cases/kubernetes.yml

```diff
@@ -0,0 +1,110 @@
+---
+parameters:
+- name: buildId
+  description: "Unique build number for each run."
+- name: PULL_BASE_REF
+  description: "Ref name of the base branch."
+- name: PULL_BASE_SHA
+  description: "Git SHA of the base branch."
+- name: PULL_REFS
+  description: "All refs to test."
+- name: PULL_NUMBER
+  description: "Pull request number."
+- name: PULL_PULL_SHA
+  description: "Pull request head SHA."
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+sync_repos:
+  - name: "aos-cd-jobs"
+actions:
+  - type: "script"
+    title: "clone k8s and check out the correct refs"
+    script: |-
+      mkdir -p src/k8s.io/kubernetes
+      cd src/k8s.io
+      git clone https://github.com/kubernetes/kubernetes.git
+      cd kubernetes
+      git checkout master
+      # Take the latest commit of the master HEAD and store it into a file
+      git rev-parse HEAD > PULL_BASE_SHA
```

Comment:
Not sure this makes sense -- if we are running as a postsubmit/periodic we aren't testing a PR (are we?) and this field is nonsense

Reply:
When the job is run periodically I can not now the commit in advance. This just reads the commit and then stores it into the ``finished.json`` file so the TestGrid knows which commit was tested.

Reply:
Hmm, yeah this seems like a reasonable workaround for this case I guess. We'd need webhooks from `k/k` to do it "right"